src/util.rs

```diff
@@ -281,9 +281,16 @@ pub mod utf8 {
 	use libc::c_char;
 
 	pub fn into<'a>(data: *const c_char, length: u32) -> Vec<&'a str> {
-		unsafe {
```

Comment:
Isn't it easier to just keep the same code but `pop()` the last element if it's an empty string?